src/main/kotlin/com/safehill/SafehillClient.kt

```diff
@@ -25,11 +27,26 @@ class SafehillClient private constructor(
     val webSocketApi: WebSocketApi,
     val currentUser: LocalUser
 ) {
+    val encryptionDetailsController by lazy {
```

Comment:
What happens when the authentication state changes? Are these classes responsive to that event?

Reply:
When the authenticationState changes, the values in the currentUser will be mutated with the new values. So, yeah.. it is responsive to that event.
But tbh, I don't like the idea that the values in the LocalUser are mutable.

Reply:
In Swift they are immutable but the reference to the user object in AppUserController changes